Userland/Libraries/LibGUI/GlyphMapWidget.cpp

```diff
@@ -49,7 +49,10 @@ void GlyphMapWidget::Selection::extend_to(int glyph)
 }
 
 GlyphMapWidget::GlyphMapWidget()
+    : AbstractScrollableWidget()
 {
+    m_original_font = font().clone();
```

Comment:
clone() can fail so the constructor isn't a good place for this. Can we check for null in paint_event() before it's de-referenced instead?

Reply:
Yep! Good catch.